sdcm/tester.py

```diff
@@ -149,6 +149,10 @@ def init_resources(self, n_db_nodes=None, n_loader_nodes=None, dbs_block_device_
             n_db_nodes = self.params.get('n_db_nodes')
         if n_loader_nodes is None:
             n_loader_nodes = self.params.get('n_loaders')
+        if n_loader_nodes <= 0:
+            self.error('Invalid parameter n_loader_nodes/n_loaders: %s,'
+                       ' it should be larger than zero.' % n_loader_nodes)
```

Comment:
Well, I never thought people would actually choose 0 for `n_loaders`. But sure, never hurts to do some sanity checking before talking to AWS. I would reckon the same should be done for `n_db_nodes`.